Mathlib/RingTheory/IntegralClosure/IsIntegralClosure/Basic.lean

```diff
@@ -356,6 +356,33 @@ theorem isIntegral_leadingCoeff_smul [Algebra R S] (h : aeval x p = 0) :
 
 end
 
+lemma quotient_isIntegralElem_of_monic (g : S[X]) (mon : g.Monic) (I : Ideal S[X]) (h : g ∈ I) :
+    ((Ideal.Quotient.mk I).comp (algebraMap S S[X])).IsIntegralElem (Ideal.Quotient.mk I X) := by
+  exact ⟨g, mon, by
+  rw [← (Ideal.Quotient.eq_zero_iff_mem.mpr h), eval₂_eq_sum_range]
+  nth_rw 3 [(as_sum_range_C_mul_X_pow g)]
+  rw [map_sum]
+  rfl⟩
```

Comment:
Why do you need the rfl?

Reply:
> Why do you need the rfl?

Lean told me to try `rfl` when I typed `apply?`. I've changed it to `simp only ...`. Thanks!